apps-rendering/src/renderer.ts

```diff
@@ -223,7 +225,7 @@ const textElement =
 						key,
 						isEditions,
 					},
-					transform(text, format),
+					children,
```

Comment:
I'm trying to understand the reason the text of the anchor doesn't need to go through transform any more? If the content has `•` or `* * *` do we not need the styling of the Bullet & HorizontalRule?

Reply:
@JamieB-gu  can probably explain it better than me, but we can get rid of the transform as I understand it, transform is evaluated in the `#text` element further up: https://github.com/guardian/dotcom-rendering/blob/df0be1b184dc51315b7146bdfbf4a6da34682927/apps-rendering/src/renderer.ts#L215-L216 so any styles added there will be inherited via the children in anchor.

Reply:
Oh I see, the content of anchor would go through recursion and end up in `#text` case which is then applied to transform function. Makes sense, thanks for explaining it :)

Reply:
Oh I see, the content of anchor would go through recursion and end up in #text case which is then applied to transform function. Makes sense, thanks for explaining it :)

Reply:
Correct yes :)